### ISSUE TITLE
\r return to start of line instead of newline

### ISSUE THREAD

**Issue by grarup** [imlinhanchao/vsc-serialport-helper#9]
In a lot of cases it would be nice if you /r did not result in a new line but just return to the begin of the current line.

E.g. printing the characters | / - \ one at a time followed by a  \r would result in a spinning character in terminal great for displaying something is alive.

This could be a setting if some people use \r as newline.

**Comment by imlinhanchao** [comment]
Are you mean the RX window of port?

**Comment by grarup** [comment]
Yes in windows showing the result received from the serail port.
